package tka_may13_Throw_Throws;

public class C5_UserException {
	public static void main(String[] args) throws C6_AgeInvalidExp{
		
		
		try {
			
			int age = -7;
			System.out.println("Exception made by user..");
			if(age > 0) {
				int useDb = age;
				System.out.println(useDb);			
				throw new NameInvalidExp();
			}
			else {
				throw new C6_AgeInvalidExp();	// [Developer]
			}
			
		}
		catch(Exception e) {
			e.printStackTrace();
		}
		
		System.out.println("thanks...");
	}
}
